common/openapi.yml

```diff
@@ -280,21 +280,21 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
-  /api/policy/v1alpha1/region:
+  /api/policy/v1alpha1/location:
     parameters:
       - $ref: '#/components/parameters/Platform'
-      - $ref: '#/components/parameters/RegionName'
+      - $ref: '#/components/parameters/GeographicLocation'
```

Comment:
I think you can use a region location here also, right? So this should not be specific to Geo?